src/app/Fake.Core.Variables/Variables.fs

```diff
@@ -0,0 +1,60 @@
+﻿/// This module contains helpers for managing build time variables
+[<RequireQualifiedAccess>]
+module Fake.Core.Variables
```

Comment:
I just fixed the compilation error and I'm not sure about the plural form. But `Variable.set` doesn't really sound better. Maybe explicitly call it `FakeVar` or `FakeContext` for example? Any idea?

Reply:
I like FakeVar

Reply:
Though that means there is a function called fakeVar so would be FakeVar.fakeVar

How about ContextVariable ?

Reply:
Yes but `FakeVar.fakeVar` ;) 
maybe `FakeVar.create` instead of that and
`FakeContext.set` for the raw stuff (or we make that internal)

Reply:
Oooo FakeVar.create love it!